Pod/Classes/SEGMixpanelIntegration.m

```diff
@@ -172,6 +172,9 @@ - (void)realTrack:(NSString *)event properties:(NSDictionary *)properties
     if (![self peopleEnabled]) {
         return;
     }
+    
+    // Handle property increments
```

Comment:
slight nit, but [comments should be sentences](https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html).